src/shared/mount-util.h

```diff
@@ -79,6 +79,11 @@ int umount_verbose(
                 const char *where,
                 int flags);
 
+int umountat_detach_verbose(
```

Comment:
typo in the commit message, it says `introduce mountat_detach_verbose()` but it's adding `umountat_detach_verbose()`